src/announce/announce.test.js

```diff
@@ -54,14 +59,29 @@ describe('bulib-announce', () => {
     expect(elementIsHidden(innerDiv)).to.be.true;
   });
 
-  it("adding 'prevent_action' with an api-enabled code stops from loading API values", async () => {
-    let el = await fixture(html`<bulib-announce debug code="primo" message="original message" prevent_action></bulib-announce>`);
-    await elementUpdated(el);
-    
-    // click on the dismiss button and wait for the element to update
+  it('renders an announcement-banner with a custom message', async () => {
+    const el = await fixture(html`<bulib-announce message="this is a custom message"></bulib-announce>`);
+    expect(el).lightDom.to.equalSnapshot();
+
     let innerMessage = el.querySelector("div.announce-banner > .message");
+    expect(innerMessage.innerText).to.include("custom message");
+  });
 
-    // assert that the innerHTML changes and that the banner becomes hidden
-    expect(innerMessage.innerText).to.include("original");
+  it('the custom message is replaced when a an api-enabled "code" is given', async () => {
+    const el = await fixture(html`<bulib-announce code="testing" message="this is a custom message"></bulib-announce>`);
+    
+    // give time for the API to get called, and then check that the message is different than the above
+    await aTimeout(800);
```

Comment:
by default, the tests fail if you go over 2 seconds. right now, we're getting kinda close to that. we'll want to change that setting.

```
Finished in 1.856 secs / 1.667 secs @ 22:42:32 GMT-0400 (Eastern Daylight Time)
```

Reply:
i believe the 2 second timeout is for each statement, so we're not too close to that limit anyway